l10n_it_fatturapa_in/models/account.py

```diff
@@ -59,7 +59,7 @@ def _compute_amount(self):
         for inv in self:
             if inv.efatt_rounding != 0:
                 inv.amount_total += inv.efatt_rounding
-                sign = inv.move_type in ["in_refund", "out_refund"] and -1 or 1
+                sign = inv.move_type in ["in_refund", "out_refund"] and 1 or -1
```

Comment:
Io francamente andrei meglio a rivedere la logica del metodo in super(). In particolare:
https://github.com/odoo/odoo/blob/14c985af526602a88666714e716283650521e537/addons/account/models/account_move.py#L848-L857

Prima cosa andrebbe usato `move.direction_sign` invece di cercare di calcolarlo noi.

Seconda cosa, di tutti campi che la `_compute_amount` riempie, vanno valutati quali cambiano. Può essere anche che in effetti non siano solo `amount_total` e `amount_total_signed`, ma anche `amount_total_in_currency_signed` da aggiornare.

Però dal codice `amount_total` è la somma delle `line.amount_currency` di ciascuna linea (valore in valuta della fattura) col `move.direction_sign`.  Invece `amount_total_signed` è la somma delle `line.balance` (valore nella valuta della company), col segno invertito per tutto ciò che non è una "entry" (è il nostro caso). E `amount_total_in_currency_signed` è ulteriormente diverso.

In più se effettivamente `amount_total_signed` è nella currency locale, l'arrotondamento andrebbe convertito prima di sommarlo.